src/preferences.h

```diff
@@ -5,18 +5,54 @@ namespace lbz
 {
 	namespace prefs
 	{
+		namespace defaults
+		{
+			static constexpr bool check_enabled = false;
+			static constexpr bool check_artist_first = true;
+			static constexpr bool check_library = false;
+			static constexpr bool check_skip = false;
+			static constexpr bool check_client_details = true;
+
+			static constexpr const char* str_user_token = "";
+			static constexpr const char* str_api_url = "https://api.listenbrainz.org/1/submit-listens";
+			static constexpr const char* str_query = "%genre% IS k-pop";
+
+			static constexpr const char* str_cache = "null";
+		}
+
 		extern cfg_bool check_enabled;
 		extern cfg_bool check_artist_first;
 		extern cfg_bool check_library;
 		extern cfg_bool check_skip;
 		extern cfg_bool check_client_details;
 
 		extern cfg_string str_user_token;
+		extern cfg_string str_api_url;
 		extern cfg_string str_query;
 
 		extern listen_cache cache;
 
 		// Legacy cache configuration, only used for migrations
 		extern cfg_string str_cache;
 	}
+
+	// Returns true, if the API URL is the default ListenBrainz service.
+	static bool is_listenbrainz()
```

Comment:
Just because my C++ is also a bit rusty, but isn't the use of `static` for a function inside a header essentially the same as `inline`? Or is there some other difference?

Reply:
I honestly don't know. Since the `is_uuid()` function is defined the same way, I just copied it.

Reply:
Good. Would be the same here :)

I *think* defining it as inline would have more or less the same effect. But really, it likely makes no difference here at all for the use case.